docs/Contributors_Guide/add_use_case.rst

```diff
@@ -1071,8 +1073,9 @@ then combine the two list items into a single item::
 
     - "met_tool_wrapper:0-54"
 
-Create a pull request from develop into develop-ref
----------------------------------------------------
+
+Update the Truth Data
+---------------------
 
 The addition of a new use case results in new output data. When this happens,
```

Comment:
George, I realize that this section is about adding a new use case. But I do think its worth noting in some way that changes to existing use cases and bugfixes can also change the output. Feel free to use the wording below... or re-word it however you see fit.

```suggestion
The addition of a new use case, changes to an existing use case, or software bug fixes can result in new or modified output data. When this happens,
```